src/specklepy/objects/geometry/box.py

```diff
@@ -0,0 +1,29 @@
+import math
+from dataclasses import dataclass
+
+from specklepy.objects.base import Base
+from specklepy.objects.geometry.plane import Plane
+from specklepy.objects.primitive import Interval
+from specklepy.objects.interfaces import IHasUnits, IHasVolume, IHasArea
+
+
+@dataclass(kw_only=True)
+class Box(Base, IHasUnits, IHasArea, IHasVolume, speckle_type="Objects.Geometry.Box"):
```

Comment:
i don't think formatting was run on this.
could you please run `pre-commit install` to make sure you have all the checks executing before a commit.